R/geom-bar.r

```diff
@@ -19,6 +19,16 @@
 #' [position_fill()] shows relative proportions at each `x` by stacking the bars
 #' and then standardising each bar to have the same height.
 #'
+#' @section Orientation:
+#' This geom treats each axis differently and can thus have two orientations.
+#' Often the orientation is easily deducable from a combination of the given
+#' mappings and the types of positional scales in use. Thus, ggplot2 will by
+#' default try to guess which orientation the layer should have. Under rare
+#' circumstances the orinetation is ambiguous and guessing may fail. In that
+#' case the orientation can be given directly using the `orientation` parameter,
+#' which can be either `"x"` or `"y"`. The value gives the axis that the geom
```

Comment:
Checking that I understand this last sentence correctly: The `orientation` value (either `"x"` or `"y"`) should be the axis you want the geom to run along. 
Right?

Reply:
yeah